index.html

```diff
@@ -144,7 +144,7 @@
 <div class="www_sectiontitle">Latest LLVM Release!</div>
 
 <div class="www_text">
-  <p><b>3 Oct 2023</b>: LLVM 17.0.2 is now <a href="releases/"><b>available
+  <p><b>3 Oct 2023</b>: LLVM 17.0.3 is now <a href="releases/"><b>available
```

Comment:
i merged too quickly, please fix the 3 Oct here

Reply:
I fixed it directly actually
https://github.com/llvm/llvm-www/commit/ebd160459c62074087c0643c0605321429918244

Reply:
Ah thanks. There is just to many manual steps to update the websites :(